components/charts/ChartBase/styles.js

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+export const SVG = styled.svg`
+  aspect-ratio: var(--aspect-ratio);
+  min-width: var(--min-width);
+  width: 100%;
+  max-height: 100%;
+  margin-inline: auto;
```

Comment:
use some old tricks to get centering + overflow when applicable + no vertical overflow